numpy/f2py/_backends/meson.build.template

```diff
@@ -0,0 +1,42 @@
+project('${modulename}',
```

Comment:
The formatting in this file is a little off - may be worth fixing in a next PR. Normal indentation is 2 or 4 spaces for function calls like `project(...)` or `py.extension_module(...)`.